.circleci/config.yml

```diff
@@ -2,42 +2,120 @@ version: 2.1
 orbs:
   coveralls: coveralls/coveralls@1.0.6
   node: circleci/node@1.1.6
+
 workflows:
-  env-vars:
+  build-test-report:
     jobs:
```

Comment:
I think we can change the order here to lint, size-check, build, test and then coverage.

Reply:
size-check depends on build because it needs to use the compiled files in `build/`, and I'm not sure what you mean by changing the order to that. You mean the `build-test-report` string?

Reply:
do those jobs start all at the same time ?